pyproject.toml

```
@@ -9,14 +9,15 @@ authors = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.11.3"
```

We may want to be a bit more conservative, and support 3.9 (as is required by `fmu-tools`

```suggestion
python = "^3.9"
```

---

fixed